res/css/structures/_RoomView.pcss

```diff
@@ -110,6 +110,17 @@ limitations under the License.
     .mx_RoomView_messagePanelSearchSpinner {
         order: 2;
     }
+
+    &[data-layout="bubble"] {
+        /* Make widgets and calls span across the whole room width when using bubble layout */
```

Comment:
This comment was taken from the title of https://github.com/matrix-org/matrix-react-sdk/pull/7553.